nuxeo-services/nuxeo-platform-convert/src/main/java/org/nuxeo/ecm/platform/convert/plugins/LibreOfficeConverter.java

```diff
@@ -67,6 +67,18 @@
         return cmdStringParameters;
     }
 
+    /**
+     * @since 11.1 If the blob is already in the format of the output, just return it without processing it.
```

Comment:
`@since` is not needed because already in the base Javadoc.
The rest of the comment is an implementation note, so should probably be just a regular comment and not a Javadoc.